docs/reference-guide/backup.md

```diff
@@ -102,3 +102,15 @@ To dump the Memgraph DB, run the following query:
 ```opencypher
 DUMP DATABASE;
 ```
+
+
+## Storage modes
+
+Memgraph has the option to work in `IN_MEMORY_ANALYTICAL` or `IN_MEMORY_TRANSACTIONAL`
+storage mode. 
+
+Memgraph starts in `IN_MEMORY_TRANSACTIONAL` mode and in such mode Memgraph offers periodic snapshots, write-ahead logging and regular snapshots as
+durability mechanism. 
+
+In `IN_MEMORY_ANALYTICAL` mode, Memgraph offers no periodic snapshots and write-ahead logging. Users can create a snapshot with the `CREATE SNAPSHOT` cypher command.
+During snapshot creation, other transactions will wait to start.
```

Comment:
Don't understand this sentence

While the snapshot is being created, other transactions will... (wait to start what where who?)

Reply:
Improved it